program/rust/src/tests/test_message.rs

```diff
@@ -38,3 +43,25 @@ fn test_twap_message_roundtrip(input: TwapMessage) -> bool {
         _ => false,
     }
 }
+
+
+fn prop_publisher_caps_message_roundtrip(input: PublisherStakeCapsMessage) -> bool {
+    let reconstructed = from_slice::<BigEndian, Message>(&input.clone().to_bytes()).unwrap();
+
+    println!("Failed test case:");
+    println!("{:?}", input);
+    println!("{:?}", reconstructed);
```

Comment:
I think we can remove these.

Reply:
Will keep since they are also in the other tests